stable/nuclio/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Nuclio release information
+nuclio:
+  version: 0.2.2
+  arch: amd64
+
+# Controller settings
+controller:
+  image: nuclio/controller
```

Comment:
Please see our standard for specifying the image.

Reply:
You're missing the point of nuclio's release and tags.
The standard is ```Docker images *should* be configurable``` which they are. But the version format nuclio are enforcing is different.

Reply:
We have a best practice for specifying the image. I don't see why you can't use it. `version` and `arch` can be combined into `tag`.